### DISCUSSION TITLE
Worked for ET2600

### DISCUSSION THREAD

**Discussion by Syriane-B** [Ircama/epson_print_conf#44] [General]
Hi all ! I just rested my ET2600 Ecotank printer Waste ink pad counter.
I've changed the pad a month ago and reset it to 80% with a free trial software.

I had some trouble to reset it with this program as it was connected through USB and not in WIFI at first. But I manage to connect it to wifi after reset it again to 80% waste with a free trial software. Actually, my printer was impossible to connect to wifi while it was in "full waste ink pad error mode". I'm glad to get it at the end! Thanks a lot for this miraculous software, bros! Cheers

**Comment by whateverusernameisnottaken** [comment]
No offense meant, but I find your post very confusing. In the end, did you reset it strictly with epson_print_conf or with another tool?

**Reply by Syriane-B** [reply]
Hi ! Yes I successfully reset my printer to 0% with this tool. You should be able to do so as well if your printer is correctly connected to wifi.

**Comment by jaque-carambole** [comment]
Hello, this software is a real gem; it allowed me to reset the waste ink tank lock on my ET2600 without any issues! My system runs on Linux Debian 12, and the ET2600 is connected via Wi-Fi. For your information, this software works fine on WINE! A big thank you to the developer!

**Reply by gegemajax037-ux** [reply]
I also successfully reset my printer to 0% (ET2600)
Thanks a lot.
